ui/pages/send/send-content/send-gas-row/send-gas-row.stories.js

```diff
@@ -23,11 +23,12 @@ export default {
   },
   argTypes: {
     insufficientBalance: {
-      name: 'Is Insufficient Balance',
       control: { type: 'boolean' },
-      defaultValue: false,
     },
   },
+  args: {
+    insufficientBalance: false,
+  },
 };
 
 export const DefaultStory = (args) => {
```

Comment:
Add recipient story works same as before. Updating arg name to reflect prop. Updating deprecated defaultValue syntax.

### Before
<img width="1440" alt="Screenshot 2022-12-11 at 7 11 54 PM" src="https://user-images.githubusercontent.com/8112138/206961087-eb08fc06-8e0c-4697-badb-3263619ecac5.png">

### After
<img width="1440" alt="Screenshot 2022-12-11 at 7 12 25 PM" src="https://user-images.githubusercontent.com/8112138/206961110-af60d99e-e48b-46b0-b281-d09c289eb11b.png">